src/main/java/com/eleks/auctioneer/controller/LotController.java

```diff
@@ -0,0 +1,42 @@
+package com.eleks.auctioneer.controller;
+
+
+import com.eleks.auctioneer.DTO.LotDTO;
+import com.eleks.auctioneer.entity.Lot;
+import com.eleks.auctioneer.service.LotService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/lot")
+public class LotController {
```

Comment:
Please, merge it with FileController
@n1kcat Please read this https://www.freecodecamp.org/news/rest-api-best-practices-rest-endpoint-design-examples/ to understand, how REST URL should be formed. @VladBrendak please help with it.